### ISSUE TITLE
sglang/python/sglang/srt/managers/scheduler.py

### ISSUE THREAD

**Issue by lbk-sys** [lbk-sys/sglang#3]
```
def run_scheduler_process(
    server_args: ServerArgs,
    port_args: PortArgs,
    gpu_id: int,
    tp_rank: int,
    pp_rank: int,
    dp_rank: Optional[int],
    pipe_writer,
    q,
):
    import torch_npu
    sender_pid = torch_npu._C._get_ipc_pid()
    q.put(sender_pid)
```
